src/main/java/apoc/custom/CypherProceduresHandler.java

```diff
@@ -598,30 +598,32 @@ private AnyValue convertToValueRecursive(Object... toConverts) {
 
     public void removeProcedure(String name) {
         withSystemDb(tx -> {
-            Node node = Iterators.single(tx.findNodes(SystemLabels.ApocCypherProcedures,
+            tx.findNodes(SystemLabels.ApocCypherProcedures,
```

Comment:
This should be not necessary.

Reply:
We can no longer assume there will no more than one result, because there are databases out in the wilds affected by #1555 that have multiple records with same name